src/components/orgAuthTokenNote.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {useLocation} from '@reach/router';
+import {graphql, StaticQuery} from 'gatsby';
+
+import {ExternalLink} from './externalLink';
+import {Note} from './note';
+import {SignedInCheck} from './signedInCheck';
+
+const siteMetaQuery = graphql`
+  query SignInNoteQuery {
+    site {
+      siteMetadata {
+        siteUrl
+      }
+    }
+  }
+`;
+
+export function OrgAuthTokenNote(): JSX.Element {
+  const location = useLocation();
+
+  return (
+    <SignedInCheck isUserAuthenticated={false}>
+      <StaticQuery
```

Comment:
Are we able to `useStaticQuery` instead of this? https://www.gatsbyjs.com/docs/how-to/querying-data/use-static-query/

Reply:
ah, yeah, much nicer!